autorest/adal/persist.go

```diff
@@ -25,6 +25,24 @@ func LoadToken(path string) (*Token, error) {
 	return &token, nil
 }
 
+// LoadCLITokens restores a set of AzureCLIToken objects from a file located at 'path'.
+func LoadCLITokens(path string) (*[]AzureCLIToken, error) {
```

Comment:
Ah, good catch @jhendrixMSFT. This program does make it look like Slices are nilable: https://play.golang.org/p/q3RBbtvzW4

Maybe that nudges us towards having it not be a pointer :)